CMakeLists.txt

```diff
@@ -31,8 +31,15 @@ set(CMAKE_CXX_STANDARD 14)
 option(NATRON_SYSTEM_LIBS "use system versions of dependencies instead of bundled ones" OFF)
 option(NATRON_BUILD_TESTS "build the Natron test suite" ON)
 
+set(IS_DEBUG_BUILD OFF)
 if(CMAKE_BUILD_TYPE MATCHES "^(debug|Debug|DEBUG)$")
+    set(IS_DEBUG_BUILD ON)
     add_definitions(-DDEBUG)
+
+    if(WIN32)
+        # Debug builds need minimal optimizations to avoid excessive link times and link errors related to Eigen.
+        add_compile_options(-O)
```

Comment:
This option is set in the qmake builds used to build the installer. That is where I got the idea to do this.

Reply:
Is it only useful on WIN32?

Reply:
It doesn't appear to be required on Linux. The Eigen linking errors I'm seeing on Windows do not appear to occur on Linux. I'm not entirely sure why.

Reply:
There are no linking issues on Linux with CMake.